packages/vscode-autohotkey2-lsp/package.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: vscode-autohotkey2-lsp
+description: Autohotkey v2 Language Support using vscode-lsp.
+homepage: https://github.com/thqby/vscode-autohotkey2-lsp
+licenses:
+  - LGPL-3.0
+languages:
+  - AutoHotKey
+categories:
+  - LSP
+
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/thqby/vscode-autohotkey2-lsp@f358179042fd653a2c536ef24bb1b03898dfd5ca
+  build:
+    - target: win
+      run: |
+        cp ./tools/install.js .
+        rm * -Exclude install.js -Recurse -Force
```

Comment:
This is a nice idea to clean up unnecessary artifacts. However, this will be executed within a git repository so removing the files will not have any meaningful impact as they're tracked within `.git/` anyway.